scio-smb/src/main/java/org/apache/beam/sdk/extensions/smb/SortedBucketIO.java

```diff
@@ -493,15 +493,9 @@ public abstract static class TransformOutput<K1, K2, V> implements Serializable
 
   /** Represents a single sorted-bucket source written using {@link SortedBucketSink}. */
   public abstract static class Read<V> implements Serializable {
-
-    @Nullable
-    abstract ImmutableList<String> getInputDirectories();
-
-    abstract String getFilenameSuffix();
-
     public abstract TupleTag<V> getTupleTag();
 
-    protected abstract BucketedInput<V> toBucketedInput(SortedBucketSource.Keying keying);
+    public abstract BucketedInput<V> toBucketedInput(SortedBucketSource.Keying keying);
```

Comment:
We can use this public method to construct SMB taps from the Scala API bindings, which take in `SortedBucketIO.Read` objects 👍